<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="inputLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="productLabel">
       <property name="text">
        <string>&amp;Product ID</string>
       </property>
       <property name="buddy">
        <cstring>productId</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="productId"/>
     </item>
     <item row="4" column="4">
      <widget class="QComboBox" name="configurationNumber"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="busLabel">
       <property name="text">
        <string>&amp;Bus number</string>
       </property>
       <property name="buddy">
        <cstring>busNumber</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QComboBox" name="deviceNumber"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="deviceLabel">
       <property name="text">
        <string>&amp;Device number</string>
       </property>
       <property name="buddy">
        <cstring>deviceNumber</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="busNumber"/>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="vendorId"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="vendorLabel">
       <property name="text">
        <string>&amp;Vendor ID</string>
       </property>
       <property name="buddy">
        <cstring>vendorId</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="configurationLabel">
       <property name="text">
        <string>&amp;Configuration</string>
       </property>
       <property name="buddy">
        <cstring>configurationNumber</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="interfaceLabel">
       <property name="text">
        <string>&amp;Interface</string>
       </property>
       <property name="buddy">
        <cstring>interfaceNumber</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QComboBox" name="interfaceNumber"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="altsettingLabel">
       <property name="text">
        <string>&amp;Alternate setting</string>
       </property>
       <property name="buddy">
        <cstring>altSetting</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QComboBox" name="altSetting"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="endpointIn">
       <property name="title">
        <string>Endpoints &quot;&amp;IN&quot;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="inLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="endpointOut">
       <property name="title">
        <string>Endpoints &quot;&amp;OUT&quot;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="outLayout">
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>RadioButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>RadioButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="descriptionTree">
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
